src/request/derive-address/DerivedIdenticonSelector.js

```diff
@@ -160,7 +166,11 @@ class DerivedIdenticonSelector extends Nimiq.Observable {
      * @private
      */
     _onSelectionConfirmed() {
-        if (!this._selectedAddress) throw new Error('Invalid state');
+        if (!this._selectedAddress) { // something went wrong
+            this._clearSelection(); // clear current selection and
+            this.nextPage(); // switch pages
+            return;
+        }
```

Comment:
Add a `TODO` that this should result in some user message that something unexpected happened, like: _Ups, something went wrong. Please try again_.